client/ngapp/build/main.js

```diff
@@ -66,6 +65,15 @@ angular.module('ooniAPIApp')
         console.log('error', error)
       })
 
+    Country.findOne({
```

Comment:
Yeah I think it's probably due to something having messed up in rebasing the branch. Let me fix this.